largestProduct.js

```diff
@@ -1,6 +1,29 @@
 class LargestProduct {
   constructor(number) {
+    this.getLargestProduct = function() {
+      let arr = ("" + number).split("");
+      return arr.reduce((accumulator, current, i, array) => {
+        let product =
+          current * array[i + 1] * array[i + 2] * array[i + 3] * array[i + 4];
+        if (product > accumulator) {
+          return product;
+        } else {
+          return accumulator;
+        }
+      }, 0);
+    };
 
+    this.getLargestProductIndex = function() {
+      let arr = ("" + number).split("");
+      let largestProduct = this.getLargestProduct();
+      for (let i = 0; i < arr.length; i++) {
```

Comment:
It looks like  you're doing twice the work here. You could just set an extra variable that lives outside the reducer but still on the object (using this) and just returned the value of this in this function.